app/controllers/api/v1/offers_controller.rb

```diff
@@ -125,6 +125,7 @@ def close_offer
     end
 
     def ggv_order_offer
+      @offer = Offer.find_by_ggv_uuid(params[:id])
       render json: @offer, serializer: serializer, exclude_messages: params[:exclude] == "messages"
     end
```

Comment:
@sunil-sharma  I think this method should move to the GoGoVanOrder controller as a 'driver_details' action. It should still return the serialized offer though.